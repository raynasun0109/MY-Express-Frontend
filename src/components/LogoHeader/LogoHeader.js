import './LogoHeader.scss';
import logo from './../../assets/MY-Shopaholic-logo.png';

export default function LogoHeader (){
    return(
        <div className="logo-container">
            <a data-testid="logo-header-button" href={`http://${window.location.host}`}>
                <img src={logo}></img>
            </a>
        </div>
    )
}