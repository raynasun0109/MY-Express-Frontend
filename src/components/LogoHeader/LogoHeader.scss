@import './../../config/_styles.scss';

.logo-container {
    width:$logo_width;
    margin:20px auto;

    img{
        width:100%;
    }
}