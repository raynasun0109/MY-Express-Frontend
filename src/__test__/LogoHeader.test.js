import renderer from 'react-test-renderer';
import LogoHeader from '../components/LogoHeader/LogoHeader.js';

it('Renders LogoHeader correctly', () => {
    const tree = renderer
      .create(<LogoHeader />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
