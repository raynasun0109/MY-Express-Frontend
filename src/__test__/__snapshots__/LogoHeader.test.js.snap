// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders LogoHeader correctly 1`] = `
<div
  className="logo-container"
>
  <a
    href="http://localhost"
  >
    <img
      src="MY-Shopaholic-big-text.png"
    />
  </a>
</div>
`;
